tasks/optional-label-check.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: optional-label-check
+spec:
+  params:
+    - name: sourceImage
+      description: the fully qualified image name
+      default: quay.io/cvpops/test-index:v4.10
+
+  steps:
+    - name: inspect-image
+      image: quay.io/rsaar/hacbs-test:latest
+      script: "skopeo inspect docker://$(inputs.params.sourceImage) > /tekton/home/image_inspect.json"
```

Comment:
No volume is defined which means that `/tekton/home/image_inspect.json` is not persisted for the next step.

Reply:
It may be saved in workspace to shared in multiple tasks (see https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md#workspaces-in-pipelines-and-pipelineruns) if we don't need to always keep the json file.

Reply:
In my image inspection tasks https://github.com/redhat-appstudio/build-definitions/pull/42, I save the json file to workspace

Reply:
We have a bit of a race condition between this PR and #42 - ideally this task would be `runAfter` the inspection task in the pipeline and would just read the inspection JSON from the workspace.

Reply:
If we want to make this task optionally standalone, we'd then first need to check if the skopeo inspection output is available in the workspace and only if it's not there run the inspection part.

Reply:
So far I merged those two steps into one, which also fixes that I forgot the shared volume. I guess that I'll wait until #42 to potentially reuse that one step?